lib/CRMS/Session.pm

```diff
@@ -31,9 +37,9 @@ sub SetupUser {
   my ($ht_user, $crms_user, $crms_id);
   my $usersql = 'SELECT id FROM users WHERE email=?';
   my $htsql = 'SELECT email FROM ht_users WHERE userid=?';
-  my $candidate = $ENV{'REMOTE_USER'};
+  my $candidate = $self->{env}->{'X-Remote-User'};
```

Comment:
I believe that an HTTP header called `X-Some-Header` will show up as `HTTP_X_SOME_HEADER` in the plack environment.

Reply:
Confirmed.

Reply:
Hoping the most recently pushed version fixes the issue.